APMrover2/motors.cpp

```diff
@@ -76,6 +76,10 @@ void Rover::output_to_motors()
         // limit throttle movement speed
         if (g.throttle_slewrate > 0) {
             SRV_Channels::limit_slew_rate(SRV_Channel::k_throttle, g.throttle_slewrate, G_Dt);
+            if (isTypeTank) {
+                // when skid steering also limit 2nd channel
+                SRV_Channels::limit_slew_rate(SRV_Channel::k_steering, g.throttle_slewrate, G_Dt);
```

Comment:
well it works on pure skid steering
If you got servo that should use k_steering you are limiting them ...